README.md

```diff
@@ -9,29 +9,12 @@ This Maven plugin provides goals to build and deploy Google App Engine applicati
 | :------------------------------ |
 | 2.0.0 has been published. The behavior of the appengine-maven-plugin has changed since v1.+; please see the [CHANGELOG](CHANGELOG.md) for a full list of changes and an updated [USER GUIDE](USER_GUIDE.md) for details. If you are having trouble using or updating your plugin, please file a [new issue](https://github.com/GoogleCloudPlatform/app-maven-plugin/issues).|
```

Comment:
Should this section say 2.1.0 now, or is it fine to leave it as 2.0.0 since it's focused on v1 -> v2 migration?

Reply:
I think I can take this down and make it a section below. It's already been like 5 months since 2.0.0

Reply:
actually, I'm just going to remove it and do nothing.